drop half-baked efforts to make things serializable #81

ThrawnCA opened the pull request.

jsotuyod commented on findbugs/src/main/java/edu/umd/cs/findbugs/BugInstance.java:
```diff
@@ -110,7 +109,7 @@
-public class BugInstance implements Comparable<BugInstance>, XMLWriteable, Serializable, Cloneable {
+public class BugInstance implements Comparable<BugInstance>, XMLWriteable, Cloneable {
 
     private static final long serialVersionUID = 1L;
```
we should drop the `serialVersionUID`s too

ThrawnCA commented:
OK, I've removed the `serialVersionUID` from the class we changed. There are a bunch of other classes that define it but aren't `Serializable`; will get to them later.

jsotuyod commented:
Awesome! Yeah, we should kill all those `serialVersionUID` and suppress `SE_BAD_FIELD` from gui components on a separate PR. Lots of dogfooding.
